.github/workflows/release-packages.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       matrix:
         include:
         - os: macos-latest
-          BUILD_OS_NAME: osx
+          BUILD_OS_NAME: macos
```

Comment:
I think this is a good change, but only to note that this value is used in the artifact naming scheme so the generated release download links will change too.

`https://github.com/etclabscore/core-geth/releases/download/v1.12.11/core-geth-osx-v1.12.11.zip` becomes `https://github.com/etclabscore/core-geth/releases/download/v1.12.11/core-geth-macos-v1.12.11.zip`

Reply:
oh, good catch. I can revert it if you want. I just changed it b/c it gave me old vibes.

Reply:
I agree that it's better to not change the naming pattern on this one.

Reply:
cool, I just changed it back. Here's the [diff](https://github.com/etclabscore/core-geth/compare/da9d92ea78fd83983a522e75768deccf5aa58a00..5b6d627954c9e2ad8d0c22fce6caaa351133265c) after the rebase